mypy: Annotate tools/deprecated/inject-messages/inject-messages.

Juanvulcano opened the pull request:
Hi there, I think I did a good job here. Getting better with each commit haha :)

smarx commented:
*Automated message from Dropbox CLA bot*

@Juanvulcano, it looks like you've already signed the Dropbox CLA. Thanks!

zbenjamin commented:
Looks good! Ping @timabbott for merge

timabbott commented on tools/deprecated/inject-messages/inject-messages:
```diff
@@ -49,20 +53,22 @@ connect_all(characters)
 connect_all(users)
 
 def truncate(string, length):
+    # type: (str, int) -> Any
```
This returns a string, right?

Juanvulcano commented:
Thanks for the feedback Tim! I knew that was a str, I had an error with strs and changed most strs to any and it stopped, I forgot to changed it lol that was embarassing. Updated 👍

showell commented:
@timabbott @Juanvulcano I think Zev and I agree that maybe we should just remove this file altogether.  (Sorry, Juan...if we do this, your work was not in vain, it helped spur the discussion.)

Juanvulcano commented:
@showell Hahaha, don't worry ! :)

zbenjamin commented:
The other file in `tools/deprecated/inject-messages` should also go. It's a copy of the play Othello.

timabbott commented:
The files have been deleted, thanks everyone!
